### PR TITLE
Add quick start & component info in readme

### PR DESCRIPTION
**Pull request by jacobras**
This change includes:

* Quick start info (about `CupertinoTheme` and `AdaptiveTheme`)
* An overview of most of the (adaptive) components to the readme
* A link to my adaptive tutorial (I can remove it if you don't want it in the readme but I figured it makes sense as it offers a step-by-step introduction to using the library based on the well known KMP starter template)
* Info about using `AdaptiveWidget` for custom widgets
* Updates the Compose badge to 1.6.0-rc01 since that's the current version (RC2 is already out btw 😀)

I took some quick screenshots of some of the components to make the page a bit more visual and lead readers to what they're looking for. They're all exported at a maximum width of 300px.

You can check what this would look like on the repo in my fork: https://github.com/jacobras/compose-cupertino/tree/docs-improvement

### CODE REVIEW

**Comment by alexzhirkevich** [comment]
Can you please move it to the separate file Adaptive.md to the docs directory and left a link to it in a readme

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Compose Cupertino
 
-[![compose-mp-version](https://img.shields.io/badge/compose--multiplatform-1.5.11-blue)](https://github.com/JetBrains/compose-multiplatform)
+[![compose-mp-version](https://img.shields.io/badge/compose--multiplatform-1.6.0--rc01-blue)](https://github.com/JetBrains/compose-multiplatform)
```
Comment (alexzhirkevich) [inline README.md:3]: Is it a Compose last released version is compiled with. Rollback it please

Comment (jacobras) [reply README.md:3]: Ah, of course. Will revert this change.

**Comment by jacobras** [comment]
> Can you please move it to the separate file Adaptive.md to the docs directory and left a link to it in a readme

Sure! Will update it today.

**Comment by jacobras** [comment]
> > Can you please move it to the separate file Adaptive.md to the docs directory and left a link to it in a readme
> 
> Sure! Will update it today.

PR updated @alexzhirkevich

**Pull request merged**